packages/csv-parser-discount-code/package.json

```diff
@@ -28,7 +28,7 @@
     "lib"
   ],
   "scripts": {
-    "build": "cross-env NODE_ENV=cli babel src --out-dir lib"
+    "build": "babel src --out-dir lib"
```

Comment:
So this would mean that in the `.babelrc` we don't get to use the `cli` environment anymore and it will default to `development` as the env key will be taken from `process.env.BABEL_ENV`, when this is not available then it uses `process.env.NODE_ENV` if even that is not available then it defaults to "development". This means we transpile with our settings for rollup builds. Which is fine and will solve the issue. However, then we can get rid of the `cli` option in the `.babelrc` overall? /cc @emmenko

Reply:
It wouldn't default to `development` as we will instead set the environment variables [here for development](https://github.com/commercetools/nodejs/pull/363/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R9) and [here for production](https://github.com/commercetools/nodejs/pull/363/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R24)